specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/examples/AscOperations_Get.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "613192d7-503f-477a-9cfe-4efc3ee2bd60",
```

Comment:
This should be all zeroes like in the other examples. Not sure whose sub ID this is.

Reply:
Fixed in next commit.